### PR TITLE
add laravel 5.8 MorphOne support

### PR DESCRIPTION
**Pull request by MonsterDong**

### CODE REVIEW

File: src/Rebing/GraphQL/Support/SelectFields.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 use Illuminate\Database\Eloquent\Relations\MorphOne;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
 use Illuminate\Database\Eloquent\Relations\MorphMany;
+use Illuminate\Database\Eloquent\Relations\MorphOne;
```
Comment (mfn) [inline src/Rebing/GraphQL/Support/SelectFields.php]: This cases travis to fail, the relation is already imported on line 16

Comment (MonsterDong) [reply src/Rebing/GraphQL/Support/SelectFields.php]: has been revised

File: src/Rebing/GraphQL/Support/SelectFields.php [hunk 2]
```diff
@@ -173,7 +173,7 @@ protected static function handleFields(array $requestedFields, $parentType, arra
                             }
                         }
                         // If 'HasMany', then add it in the 'with'
-                        elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, MorphOne::class))
+                        elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphOne::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, MorphOne::class))
```
Comment (mfn) [inline src/Rebing/GraphQL/Support/SelectFields.php]: I don't understand this change, `MorphOne` _is already_ part of the `elseif` check (it's at the end), see the original line (emphasis mine):
>  elseif ((is_a($relation, HasMany::class) || is_a($relation, MorphMany::class) || is_a($relation, HasOne::class) || is_a($relation, **MorphOne::class**))

Can you elaborate how it fixes it and what exactly is broken with this change?

**Comment by MonsterDong** [comment]
我的版本可能比较落后

**Pull request closed without merging**